utils/mock-data-loader/banks/index.js

```diff
@@ -159,6 +159,36 @@ const MOCK_BANKS = [
     ],
     isVisibleInTfmUtilisationReports: true,
   },
+  {
+    id: '12',
+    name: 'ICICI',
+    mga: ['Test.pdf'],
+    emails: ['maker1@ukexportfinance.gov.uk'],
+    companiesHouseNo: 'FC011846',
+    partyUrn: '00313097',
+    hasGefAccessOnly: false,
+    paymentOfficerTeam: {
+      teamName: 'Payment Reporting Team',
+      emails: ['payment-officer4@ukexportfinance.gov.uk'],
+    },
+    utilisationReportPeriodSchedule: MONTHLY_REPORT_PERIOD_SCHEDULE,
+    isVisibleInTfmUtilisationReports: true,
+  },
+  {
+    id: '12',
```

Comment:
Utilisation reports tests are failing because we have to insert a report for each bank opted in to utilisation reporting and we make the assumption that bank ids will be unique across banks. I think that assumption is correct and this needs to be different to the id of the bank above.